### PR TITLE
Update Onboarding Privacy Copy (IOS-221)

### PR DESCRIPTION
**Pull request by kimar**
# Rationale

| Screenshot | Description |
|---|---|
| Simulator Screenshot - iPhone 15 - 2024-05-18 at 14 13 51 | This PR modifies the Onboarding Privacy Copy and allows it to render Markdown. |

### CODE REVIEW

File: Mastodon/Scene/Onboarding/Privacy/PrivacyTableViewController.swift [hunk 1]
```diff
@@ -135,23 +136,37 @@ extension PrivacyTableViewController: UITableViewDelegate {
...
+            wrapper.bottomAnchor.constraint(equalTo: label.bottomAnchor, constant: 16)
         ])
 
         return wrapper
     }
 }
 
 extension PrivacyTableViewController: OnboardingViewControllerAppearance { }
+
+private struct HeaderTextView: View {
```
Comment (zeitschlag) [inline Mastodon/Scene/Onboarding/Privacy/PrivacyTableViewController.swift:163]: Please forgive me the stupid question, but what is the advantage of using a SwiftUI-view here over using an [attributed string](https://developer.apple.com/documentation/foundation/nsattributedstring/3796597-init)?

**Review by zeitschlag — APPROVED** [review]
No remarks except the question, so 👍

File: Mastodon/Scene/Onboarding/Privacy/PrivacyTableViewController.swift (hunk 1 above)
Comment (DK1DA) [reply Mastodon/Scene/Onboarding/Privacy/PrivacyTableViewController.swift:163]: Not a stupid question at all! The problem was that, using the NSAttributedString, the bold markdown looked much different and I couldn't figure out why...

Bildschirmfoto 2024-05-17 um 15 17 39

Comment (DK1DA) [reply Mastodon/Scene/Onboarding/Privacy/PrivacyTableViewController.swift:163]: Also the line breaks inside the markdown were not rendered correctly 🤷

**Pull request merged**